resources/config/routing.yml

```diff
@@ -258,3 +257,13 @@ privacy:
    methods: [GET]
    controller: OpenCFP\Http\Action\Signup\PrivacyAction
 
+central:
+  path: /central
+  methods: [GET]
+  controller: OpenCFP\Http\Action\Security\CentralAction
+
+redirect:
+  path: /redirect
```

Comment:
These paths seem a bit generic, perhaps it makes sense to rename a few of the paths?

* `/opencfp-central`
* `/opencfp-central/redirect`

or

* `/sso`
* `/sso/redirect`

What do you think?

In the same light, perhaps renaming the actions also makes sense?

Reply:
I have made those suggested changes, need to also update the wording on the OpenCFP Central web site to reflect this.